Bloggist/src/triggers/statusDraftTrigger.trigger

```diff
@@ -0,0 +1,5 @@
+trigger statusDraftTrigger on Post__c (before insert) {
+    for(Post__c p : Trigger.New) {
+       p.Status__c = 'Draft';
```

Comment:
Why is this trigger here?

Reply:
This was a first practical attempt of using triggers on this exercise, the same way as in the above comment i was managing the status changes with triggers.  Personally i though it was a good idea to use them here to practice but the status changes will be managed in another way if the impact on performance or something is better.